volume/local/local_unix.go

```diff
@@ -71,6 +69,29 @@ func (v *localVolume) mount() error {
 	if v.opts.MountDevice == "" {
 		return fmt.Errorf("missing device in volume options")
 	}
-	err := mount.Mount(v.opts.MountDevice, v.path, v.opts.MountType, v.opts.MountOpts)
+	r1, _ := regexp.Compile(`\baddr=*`)
+	mountOpts := v.opts.MountOpts
+	if r1.MatchString(v.opts.MountOpts) && v.opts.MountType == "nfs" {
+		if addrValue := getAddress(v.opts.MountOpts); addrValue != "" && net.ParseIP(addrValue).To4() == nil {
+			ipAddr, err := net.ResolveIPAddr("ip", addrValue)
+			if err != nil {
+				return errors.Wrapf(err, "Unable to resolve hostname %s", addrValue)
```

Comment:
This wrap message is not really adding any context and just duplicating what is already there since the error from `net` will be `lookup <addrValue>: no such host`.

Maybe something like `error resolving passed in nfs address`.
Also, errors should always be started as lower case.